openapi/src/main/java/io/micronaut/openapi/visitor/ElementUtils.java

```diff
@@ -219,12 +221,16 @@ public static boolean isIgnoredParameter(TypedElement parameter) {
             || parameter.isAnnotationPresent(Hidden.class)
             || parameter.isAnnotationPresent(JsonIgnore.class)
             || parameter.booleanValue(Parameter.class, "hidden").orElse(false)
-            || isAnnotationPresent(parameter, "io.micronaut.session.annotation.SessionValue")
-            || isAnnotationPresent(parameter, "org.springframework.web.bind.annotation.SessionAttribute")
-            || isAnnotationPresent(parameter, "org.springframework.web.bind.annotation.SessionAttributes")
+            || isParamAnnotationPresent(parameter, "io.micronaut.session.annotation.SessionValue")
+            || isParamAnnotationPresent(parameter, "org.springframework.web.bind.annotation.SessionAttribute")
+            || isParamAnnotationPresent(parameter, "org.springframework.web.bind.annotation.SessionAttributes")
             || isIgnoredParameterType(parameter.getType());
     }
 
+    private static boolean isParamAnnotationPresent(Element element, String className) {
+        return element.findAnnotation(className).isPresent();
```

Comment:
There is not hasAnnotation

Reply:
@dstepanov ??? What do you mean? How I need to change it?

Reply:
do I need to change it  to this:
```java
parameter.isAnnotationPresent("org.springframework.web.bind.annotation.SessionAttribute")
```
?

Reply:
and what difference between `hasAnnotation` and `isAnnotationPresent` ?

Reply:
I think it’s better to avoid returning an optional

Reply:
I'm sorry, I still didn't get it.: Do I need to change something? If so, what should I change?

Reply:
@dstepanov ok, I changed it to `hasAnnotation`. But what difference between `hasAnnotation` and `isAnnotationPresent` ?